<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mode_pidt_tw</class>
 <widget class="QWidget" name="Mode_pidt_tw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_change_mode_pidt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode PIDT</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Velocity Control</string>
   </property>
   <widget class="QDoubleSpinBox" name="sb_pid_vel_kp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_80">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KI:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_vel_ki">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_81">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KD:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_vel_kd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_82">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_vel_kp_step">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_83">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_vel_ki_step">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_84">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_vel_kd_step">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mode_pidt_write_params_pidt_vel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Tilt Control</string>
   </property>
   <widget class="QDoubleSpinBox" name="sb_pid_tilt_kp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_85">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_86">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KI:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_tilt_ki">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_87">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KD:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_tilt_kd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_88">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_tilt_kp_step">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_89">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_tilt_ki_step">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_90">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_tilt_kd_step">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mode_pidt_write_params_pidt_tilt">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="cb_vel_pos_pidt">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Velocity Control</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Position Control</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Position Control</string>
   </property>
   <widget class="QDoubleSpinBox" name="sb_pid_pos_kp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_91">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_92">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KI:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_pos_ki">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_93">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KD:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_pos_kd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_94">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_pos_kp_step">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_95">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_pos_ki_step">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_96">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_pid_pos_kd_step">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mode_pidt_write_params_pidt_pos">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_mode_pidt_save_params">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mode_pidt_load_params">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mode_pidt_write_params">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>621</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VX:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_pidt_vx">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>W:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_pidt_w">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_control_enable">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
